src/core/filesystem.cc

```diff
@@ -27,13 +27,16 @@
 #include "src/core/filesystem.h"
 
 #include <dirent.h>
+#include <google/cloud/storage/client.h>
```

Comment:
Need ifdef TRTIS_ENABLE_GCS to protect the parts of the file that depend on GCS. You can do a build to test and make sure you did it correctly.